.vale/styles/RedHat/Using.yml

```diff
@@ -0,0 +1,14 @@
+---
+extends: sequence
+message: "Use 'by using' instead of 'using' when it follows a noun for clarity and grammatical correctness."
+link: https://redhat-documentation.github.io/vale-at-red-hat/docs/main/reference-guide/using/
+level: warning
+action:
+  name: edit
+  params:
+    - regex
```

Comment:
Just checking the regex param is required?

Reply:
It does look funny but yes it needs to be there. At least in my testing. https://vale.sh/docs/topics/actions/#regex